utils/client.tsx

```diff
@@ -0,0 +1,6 @@
+import Stripe from 'stripe';
+
+// Use SDK to connect to stripe using my SECRET
+export const stripeClient = new Stripe(process.env.STRIPE_SECRET_KEY!, {
+  apiVersion: '2022-11-15',
+});
```

Comment:
maybe this file should be named `util/stripe.ts`, 3 changes:

- `util` intead of `utils`
- `stripe` instead of `client`
- `ts` instead of `tsx`